src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -218,6 +218,8 @@ let run ~logger ~trust_system ~verifier ~network ~consensus_local_state
            , `Overflow
                (Drop_head
                   (fun head ->
+                    Mina_metrics.(
+                      Counter.inc_one Pipe.Overflow.bootstrap_sync_ledger) ;
```

Comment:
Nit: it might be easier to understand the metric at a glance if it had `drops` in the name